app/src/main/AndroidManifest.xml

```diff
@@ -4,9 +4,9 @@
     <uses-permission android:name="android.permission.INTERNET"/>
     <application
         android:allowBackup="true"
-        android:icon="@mipmap/ic_launcher"
+        android:icon="@drawable/ic_mealplanner"
         android:label="@string/app_name"
-        android:roundIcon="@mipmap/ic_launcher_round"
+        android:roundIcon="@drawable/ic_mealplanner"
```

Comment:
https://cdn.discordapp.com/attachments/512657675016863744/826627549957324840/image2.PNG

This give it a little bit more life, you can this to whatever you like, this create from the image vector using android studio.